packages/elements/src/combo-box/__demo__/index.html

```diff
@@ -170,10 +170,10 @@
 
           // return scoped custom filter
           return (item) => {
+            const value = item.value;
+            const label = item.label;
             const regex = getRegularExpressionOfQuery();
-            // test on label or value
-            const result = query === item.value || regex.test(item.label);
-            regex.lastIndex = 0; // do not forget to reset last index
+            const result = regex.test(value) || regex.test(label);
```

Comment:
I am just curious why demo in v7 doesn't require to reset lastIndex but v6 does

Reply:
Nice catch! They are all removed. Note that, it applies to both versions.